internal/service/cloudbackupsnapshotexportbucket/data_source_cloud_backup_snapshot_export_buckets.go

```diff
@@ -95,7 +97,7 @@ func dataSourceRead(ctx context.Context, d *schema.ResourceData, meta any) diag.
 	return nil
 }
 
-func flattenBuckets(buckets []admin.DiskBackupSnapshotExportBucket) []map[string]any {
+func flattenBuckets(buckets []admin.DiskBackupSnapshotExportBucketResponse) []map[string]any {
```

Comment:
change from the SDK, no impact

Reply:
I think [this context](https://mongodb.slack.com/archives/CLB2CJ1JB/p1729539408447839) can be useful

Reply:
nice, thank you!